jwst/assign_wcs/tests/test_nirspec.py

```diff
@@ -57,7 +57,7 @@ def create_hdul():
     phdu.header['instrume'] = 'NIRSPEC'
     phdu.header['detector'] = 'NRS1'
     phdu.header['time-obs'] = '8:59:37'
-    phdu.header['date-obs'] = '2014-09-05'
+    phdu.header['date-obs'] = '2016-09-05'
```

Comment:
I don't expect these tests to pass because the data used for comparison is pre-CV3. As soon as we see that they fail and don't error (using CV3 reference files) they can be x-failed. The comparison data will be updated later in a separate PR.
The `correct_tilt` test failure indicates pre-CV3 reference files are used.